tests/export_test.py

```diff
@@ -41,6 +40,14 @@ def test_export_open_hdf5(ds_local):
     ds_opened = vaex.open(filename)
     assert list(ds) == list(ds_opened)
 
+def test_export_open_csv(ds_local):
+    df = ds_local
+    df = df.drop(df.obj)
+    filename = tempfile.mktemp(suffix='.csv')
```

Comment:
Please use the pytest tmpdir fixture, you can find examples in this source file

Reply:
Done!
(I saw there are tests with and without this.. should we eventually switch to tmpdir for everything?)

Reply:
I think in most cases we should.